pyproject.toml

```diff
@@ -56,6 +56,7 @@ dependencies = [
     "unyt>=2.9.2",
     "tomli>=1.2.3;python_version < '3.11'",
     "typing-extensions>=4.4.0;python_version < '3.12'",
+    "rlic>=0.3.2",
```

Comment:
On my phone at the moment, so note to self:
I should move this above other other unconditional dependencies. I also want to make sure the minimal version is correctly pinned on minimal env jobs.